components/calendly_v2/actions/get-event/get-event.mjs

```diff
@@ -14,21 +25,37 @@ export default {
     },
     event_uuid: {
       type: "string",
-      description: "The event's unique identifier.",
+      label: "Event UUID",
+      description:
+        "The event's unique identifier.",
+      optional: true,
+    },
+    event_url: {
+      type: "string",
+      label: "Event URL",
+      description: "The URL of the event to retrieve information about. If you are using a Calendly Source in the same workflow, you would use *{{steps.trigger.event.payload.event}}*.",
+      optional: true,
     },
   },
-  async run({ $ }) {
-  //See the API docs: https://developer.calendly.com/docs/api-docs/reference/calendly-api/openapi.yaml/paths/~1scheduled_events~1%7Buuid%7D/get
 
-    if (!this.event_uuid) {
-      throw new Error("Must provide event_uuid parameter.");
+  async run({ $ }) {
+    if (!this.event_uuid && !this.event_url) {
+      throw new ConfigurationError(
+        "Error: You must provide either Event UUID or Event URL.",
```

Comment:
It's already clear that it's an error, you can remove the leading "Error: ", and I'd modify to something more friendly, like "Please provide either the Event UUID or Event URL, then try again"